Doc/library/textwrap.rst

```diff
@@ -57,14 +95,22 @@ functions should be good enough; otherwise, you should use an instance of
       >>> textwrap.shorten("Hello world", width=10, placeholder="...")
       'Hello...'
 
-   Optional keyword arguments correspond to the instance attributes of
-   :class:`TextWrapper`, documented below.  Note that the whitespace is
-   collapsed before the text is passed to the :class:`TextWrapper` :meth:`fill`
-   function, so changing the value of :attr:`.tabsize`, :attr:`.expand_tabs`,
-   :attr:`.drop_whitespace`, and :attr:`.replace_whitespace` will have no effect.
+   See :class:`TextWrapper` for documentation on all additional optional
+   arguments:
 
-   .. versionadded:: 3.4
+   * :attr:`.initial_indent`
+   * :attr:`.subsequent_indent`
+   * :attr:`.fix_sentence_endings`
+   * :attr:`.break_long_words`
+   * :attr:`.break_on_hyphens`
+   * :attr:`.max_lines`
 
+   Note that :class:`TextWrapper` has some additional options which have **no
```

Comment:
I think it's good to keep the old version of this paragraph (starting at 'Note that whitespace ...') because it explains the reason why the args are no-op.

Reply:
Ok, I restored it. See c346c1d04863174c1a0ff5437cf45f87b1fc5545